client/src/components/game/gui/meters/Meters.jsx

```diff
@@ -80,8 +79,12 @@ function Meters() {
                 setMaxEnergy(data.new);
             }),
             PubSub.subscribe(COMBAT_STATUS_TRIGGER, (msg, data) => {
+                data = parseInt(data, 10);
+                if (data.isNaN) {
```

Comment:
I think this should be `Number.isNaN(data)`.
`data.isNaN` doesn't look like a valid property.